BackofficeBundle/Resources/views/BackOffice/Include/mainPanel.html.twig

```diff
@@ -2,10 +2,13 @@
 <div id="main" role="main">
 
     {% include 'OpenOrchestraBackofficeBundle:BackOffice/Include/MainPanel:ribbon.html.twig' %}
-
+	{% set alert %}
+	    {% include 'BraincraftedBootstrapBundle::flash.html.twig' %}
+	{% endset %}
```

Comment:
@itkg-ngilain , we are in a context of js break and form submit with whole page refresh.